tests/Fixtures/project/src/Aspect/InitializationAspect.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Go\Tests\TestProject\Aspect;
+
+use Go\Aop\Aspect;
+use Go\Lang\Annotation as Pointcut;
+
+class InitializationAspect implements Aspect
+{
+    /**
+     * Before class instance initialization.
+     *
+     * @Pointcut\Before("initialization(Go\Tests\TestProject\Application\Main)")
+     */
+    public function beforeInstanceInitialization()
+    {
+        echo 'It invokes before class instance is initialized.';
+    }
+
+    /**
+     * After class initialization.
+     *
+     * @Pointcut\Before("staticinitialization(Go\Tests\TestProject\Application\Main)")
```

Comment:
Comment name and method name "After" differs from the actual pointcut type, which is "Before"